lib/ansible/modules/pip.py

```diff
@@ -350,24 +351,56 @@ def _get_cmd_options(module, cmd):
     return cmd_options
 
 
-def _get_packages(module, pip, chdir):
-    '''Return results of pip command to get packages.'''
+def _get_packages(module, pip, chdir, env):
+    '''Return the installed packages as a string, one package per line, like:
```

Comment:
Let's follow PEP 257 here
```suggestion
    '''Return the installed packages as a string.
    
    The result is formatted as one package per line, like:
```